packages/react-native-payments-addon-stripe/README.md

```diff
@@ -13,6 +13,12 @@ Second, install the [React Native Payments CLI](https://www.npmjs.com/package/re
 $ yarn add react-native-payments-cli
 ```
 
+Then, complete the Carthage dependency. Otherwise the next step fails:
+```bash
+$ cd node_modules/react-native-payments-addon-stripe
+$ carthage update
+```
+
```

Comment:
Could you please mention Braintree addon as well?

Reply:
And to be more precise, addon itself is working as expected.
The issue here is NPM auto pruning Carthage generated files. Change you propose fixes the issue until very next `npm install {package}` or `yarn add {package}`. We should mention this as well.